### DISCUSSION TITLE
Print crashing

### DISCUSSION THREAD

**Discussion by madyury007** [Pierro55/Loop#6] [General]
Hello Pierro, thank you for designing the LOOP. So I printed the legs and added to one of my printers, when I print my usual part print often crashes few times mid print. It never crashed without the legs. I print with PETG, 0.6mm nozzle. Do you have any suggestions?

**Comment by Pierro55** [comment]
@madyury007 Sounds like you are printing with crash detection enabled. For Loop to work properly you have to disable crash detection as mentioned in the instructions.
Hope that helps! :)
